ui/raidboss/data/06-ew/dungeon/aloalo_island.ts

```diff
@@ -187,71 +155,70 @@ const triggerSet: TriggerSet<Data> = {
       infoText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Adds',
-          de: 'Adds',
-          ja: 'ざこギミック',
+          en: 'Bubbles Move 2 Tiles',
+          de: 'Blasen 2 Flächen Rückstoß',
+          ja: '泡のみ2マスのノックバック',
         },
       },
     },
     {
-      id: 'Aloalo Ketuduke Fluke Typhoon',
-      type: 'StartsUsing',
-      netRegex: { id: '8A84', source: 'Ketuduke', capture: false },
+      // Path 01
+      id: 'Aloalo Ketuduke Water III',
+      type: 'Tether',
+      netRegex: { id: '0001', source: 'Summoned Apa' },
+      condition: Conditions.targetIsYou(),
       infoText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Bubbles 2 tiles knockback',
-          de: 'Blasen 2 Flächen Rückstoß',
-          ja: '泡のみ2マスのノックバック',
+          en: 'Stretch Tether',
         },
       },
     },
     {
-      id: 'Aloalo Ketuduke Updraft',
+      // Path 02
+      id: 'Aloalo Ketuduke Tidal Wave',
       type: 'StartsUsing',
-      netRegex: { id: '8D0F', source: 'Ketuduke', capture: false },
-      infoText: (_data, _matches, output) => output.text!(),
+      netRegex: { id: '8D12', source: 'Ketuduke', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Bubbles float',
-          de: 'Blasen schweben',
-          ja: '泡のみ浮上',
+          en: 'Knockback => Get in Bubble',
         },
       },
     },
     {
-      id: 'Aloalo Ketuduke Hydrobomb',
-      type: 'StartsUsing',
-      netRegex: { id: '8D0F', source: 'Ketuduke', capture: false },
-      infoText: (_data, _matches, output) => output.text!(),
+      // Path 03
+      id: 'Aloalo Ketuduke Zaratan Add',
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '12539', capture: false },
+      condition: (data) => data.isKetuFight,
+      delaySeconds: 16,
+      suppressSeconds: 5,
+      alertText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Bubbles 2 tiles knockback + Puddles',
-          de: 'Blasen 2 Flächen Rückstoß + Flächen',
-          ja: '泡のみ2マスのノックバック + ゆか',
+          en: 'Get behind non-bubbled Zaratan',
         },
       },
     },
     {
-      id: 'Aloalo Ketuduke Hydrobomb Chasing',
-      type: 'Ability',
-      netRegex: { id: '8D0F', source: 'Ketuduke', capture: false },
-      delaySeconds: 1.5,
-      infoText: (_data, _matches, output) => output.text!(),
+      // Path 04
+      id: 'Aloalo Ketuduke Ogrebon Add',
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '12540', capture: false },
+      condition: (data) => data.isKetuFight,
+      delaySeconds: 18,
+      suppressSeconds: 5,
+      alertText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Avoid Chasing AOEs',
```

Comment:
I think the Chasing AOE puddles are not worth calling out here. There's no headmarker on players (so it's not targeted on somebody directly) and it's on the timeline too.